### DISCUSSION TITLE
Can Blitz Recipes be used with a non Blitz project?

### DISCUSSION THREAD

**Discussion by mbrain-io** [blitz-js/blitz#3488] [Ideas] [answered]
Blitz Recipes are awesome, but what if I want to use them on pure NextJS projects? Is it possible?

**Comment by beerose** [comment, marked as answer]
Hey! Currently, it's not possible as most of the recipes rely on Blitz stuff. Some of them assume that you're in a Blitz project and others import things from Blitz and/or modify Blitz config. However, as we're moving to a framework agnostic toolkit, we want the recipes to work on pure Nextjs projects too.
